Gemfile

```diff
@@ -1,6 +1,11 @@
 source 'https://rubygems.org'
 
 group :test do
+  gem 'rspec', group: :test 
+  gem 'pg'
+  gem 'sinatra'
+  gem 'capybara', group: :test
+  gem 'selenium-webdriver'
   gem 'rspec'
   gem 'rubocop', '0.71.0'
   gem 'simplecov', require: false
```

Comment:
I think it would be worth trying to tidy up this list, ie. delete duplicate rspec, delete` group: :test` reference for rspec and capybara as they are already incuded in "global" `group :test`